package.json

```diff
@@ -0,0 +1,89 @@
+{
+  "name": "zonos-typescript-sdk",
+  "version": "0.0.1",
+  "private": true,
+  "dependencies": {
+    "graphql": "^16.6.0",
+    "graphql-request": "^4.3.0",
+    "graphql-tag": "^2.12.6"
+  },
+  "engines": {
+    "node": "18"
+  },
```

Comment:
What do you think about removing the consuming projects to a particular node version?
```suggestion
```

Reply:
It's removed in the packed version, so the consuming project will not be limited to the node version. I just restrict it for whoever developing this project.